tokens/tori.fi/foundation.yml

```
@@ -0,0 +1,15 @@
+---
+token: maps
+
+text:
+  color:
+    _: red-700
+    subtle: gray-500
+    placeholder: bluegray-300
+    inverted:
+      _: white-100
```

Should this be `white`?
```suggestion
      _: white
```

---

Thanks! Fixed all cases in [1ebe4a4](https://github.com/warp-ds/tokens/pull/18/commits/1ebe4a4ad53d769b506a349b6827fce12ee01970)